python/cugraph/structure/graph.py

```diff
@@ -565,945 +277,394 @@ def from_numpy_matrix(self, np_matrix):
-        undirected edgelist for a Graph the upper trianglar matrix of the
-        symmetrized edgelist is returned.
-
-        networkx.Graph renumbers the input and stores the upper triangle of
-        this renumbered input. Since the internal renumbering of networx and
-        cugraph is different, the upper triangular matrix of networkx
-        renumbered input may not be the same as cugraph's upper trianglar
-        matrix of the symmetrized edgelist. Hence the displayed source and
-        destination pairs in both will represent the same edge but node values
-        could be swapped.
-
+        df: cudf.DataFrame or dask_cudf.DataFrame
+            A DataFrame containing internal vertex identifiers that will be
+            converted into external vertex identifiers.
+        column_name: string
+            Name of the column containing the internal vertex id.
+        preserve_order: (optional) bool
+            If True, preserve the order of the rows in the output
+            DataFrame to match the input DataFrame
         Returns
-        -------
-        df : cudf.DataFrame
-            This cudf.DataFrame wraps source, destination and weight
-
-            df[src] : cudf.Series
-                contains the source index for each edge
-            df[dst] : cudf.Series
-                contains the destination index for each edge
-            df[weight] : cusd.Series
-                Column is only present for weighted Graph,
-                then containing the weight value for each edge
-        """
-        if self.distributed:
-            if self.edgelist is None:
-                raise Exception("Graph has no Edgelist.")
-            return self.edgelist.edgelist_df
-        if self.edgelist is None:
-            src, dst, weights = graph_primtypes_wrapper.view_edge_list(self)
-            self.edgelist = self.EdgeList(src, dst, weights)
-
-        edgelist_df = self.edgelist.edgelist_df
-
-        if self.renumbered:
-            edgelist_df = self.unrenumber(edgelist_df, "src")
-            edgelist_df = self.unrenumber(edgelist_df, "dst")
-
-        if type(self) is Graph or type(self) is MultiGraph:
-            edgelist_df = edgelist_df[edgelist_df["src"] <= edgelist_df["dst"]]
-            edgelist_df = edgelist_df.reset_index(drop=True)
-            self.edge_count = len(edgelist_df)
-
-        return edgelist_df
-
-    def delete_edge_list(self):
-        """
-        Delete the edge list.
+        ---------
+        df : cudf.DataFrame or dask_cudf.DataFrame
+            The original DataFrame columns exist unmodified.  The external
+            vertex identifiers are added to the DataFrame, the internal
+            vertex identifier column is removed from the dataframe.
         """
-        # decrease reference count to free memory if the referenced objects are
-        # no longer used.
-        self.edgelist = None
+        return self.renumber_map.unrenumber(df, column_name, preserve_order)
```

Comment:
`self.renumber_map` no longer exists in the `graph` class. Maybe a workaround could be `self.Impl.renumber_map.unrenumber` assuming `self.Impl` is not `None`  ?

Reply:
https://github.com/rapidsai/cugraph/pull/1404#discussion_r604365353